deploy-board/deploy_board/webapp/util_views.py

```diff
@@ -39,11 +39,17 @@ def _convert_opentsdb_data(dps):
 
 def _get_latest_metrics(url):
     response = urllib2.urlopen(url)
-    data = json.loads(response.read())
+    decoded_response = str(response.read()).decode('string_escape')
+    data = json.loads(decoded_response)
+
     # Return the first datapoint in the datapoints list
     if data:
+        if 'data' in data:
```

Comment:
This put the TSDB case in risk. I'd avoid this and do something like 
`if 'data' in data and 'datapoints' in data['data'][0] and len(data['data'][0]['datapoints']) > 0:`

Reply:
Or even better
```python
try:
  return [dp for dp in data['data][0]['datapoints'] if dp[1] != None]
except:
  pass
```